app/Http/Controllers/UserCreditsController.php

```diff
@@ -9,7 +9,7 @@
 
 class UserCreditsController extends Controller
 {
-    public function store(Request $request, $id){
+    public function store(Request $request, $id) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->